docs/monoDrive_home/IMU.md

```diff
@@ -1,39 +1,34 @@
 # IMU
 
-Provides inertial measurements for the ego vehicle relative to the sensor's location.
+The monoDrive IMU sensor reports the ego vehicle's acceleration in x, y and z 
+axis, as well as the angular velocity in the same axis. The location and 
+orientation of the sensor can be modified in the "x", "y" and "z" axis with 
+respect to the origin of the ego vehicle.
 
 ## Configuration
 
-```
-[
-  {
+```json
+{
     "type": "IMU",
-    "listen_port": 8500,
+    "listen_port": 0,
```

Comment:
Fix the listen port, we probably want to set these to whatever the LabVIEW client is setting them to at the moment.